cmd/main.go

```diff
@@ -44,6 +51,9 @@ func main() {
 	gitLabWrapper := wrappers.NewGitLabWrapper()
 	bflWrapper := wrappers.NewBflHTTPWrapper(bfl)
 
+	kicsContainerID := uuid.New()
+	viper.Set(params.KicsContainerNameKey, kicsContainerPrefixName+kicsContainerID.String())
```

Comment:
@tiagobcx  could we create this only inside the kics command ?

Reply:
yes, done